src/measure_theory/integral/lebesgue.lean

```diff
@@ -2011,10 +2023,15 @@ begin
     simp [lintegral_supr, ennreal.mul_supr, h_mf.mul (h_mea_g _), *] }
 end
 
+lemma set_lintegral_with_density_eq_set_lintegral_mul (μ : measure α) {f g : α → ℝ≥0∞}
+  (hf : measurable f) (hg : measurable g) {s : set α} (hs : measurable_set s) :
+  ∫⁻ x in s, g x ∂μ.with_density f = ∫⁻ x in s, (f * g) x ∂μ :=
+by rw [restrict_with_density hs, lintegral_with_density_eq_lintegral_mul _ hf hg]
+
 /-- In a sigma-finite measure space, there exists an integrable function which is
 positive everywhere (and with an arbitrarily small integral). -/
-lemma exists_pos_lintegral_lt_of_sigma_finite
-  {α} [measurable_space α] (μ : measure α) [sigma_finite μ] {ε : ℝ≥0∞} (ε0 : ε ≠ 0) :
+lemma exists_integrable_pos_of_sigma_finite
+  (μ : measure α) [sigma_finite μ] {ε : ℝ≥0} (εpos : 0 < ε) :
```

Comment:
You should use the `(ε0 : ε ≠ 0)` hypothesis instead of `(εpos : 0 < ε)`. Also it looks like you changed the name of the lemma in the merge.

Reply:
I pressed the accept change in the change without much thought. Thanks!